include/jwt-cpp/jwt.h

```diff
@@ -70,10 +70,10 @@ namespace jwt {
 	namespace algorithm {
 
 		struct none {
-			std::string sign(const std::string& data) const {
+			std::string sign(const std::string&) const {
```

Comment:
Would you prefer I just comment out the name instead of removing it?

Reply:
I think it is fine to remove it.